### PR TITLE
[zh-cn] improve the translation about /zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html

### PR DESCRIPTION
**Pull request by qlijin**
improve the translation about this file's two place

### CODE REVIEW

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -26,7 +26,8 @@
         <div class="row">
 
      <div class="col-md-8">
-          <h3>Objectives</h3>
+<!--        <h3>Objectives</h3>-->
```
Comment (Sea-n) [inline content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html:29]:
```suggestion
          <!-- <h3>Objectives</h3> -->
```

**Review by Sea-n — COMMENTED** [review]
/approve

File: content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html (hunk 1 above)
Comment (qlijin) [reply content/zh-cn/docs/tutorials/kubernetes-basics/update/update-intro.html:29]: OK

**Pull request merged**